packages/blocks-ui/package.json

```diff
@@ -22,14 +22,17 @@
     "@theme-ui/components": "^0.2.50",
     "@theme-ui/editor": "^0.2.51",
     "@theme-ui/presets": "^0.2.44",
+    "add": "^2.0.6",
```

Comment:
I've done this before. My guess is you've ran `yarn add yarn add react-loadable` on accident, so it installed `add` and `yarn` as dependencies. 😂

Reply:
LOL 🤣